tests/test_utils.py

```diff
@@ -172,3 +176,59 @@ def test_p_values_from_sample():
 
     ks_results = insertion_p_value(ns.insertion[nlive:-nlive], nlive, batch=1)
     assert ks_results['p-value'] > 0.05
+
+
+def test_credibility_interval():
+    np.random.seed(0)
+    from anesthetic import read_chains
+    normal_samples = np.random.normal(loc=2, scale=0.1, size=10000)
+    mean, cov = credibility_interval(normal_samples, level=0.68,
+                                     return_covariance=True)
+    assert_allclose(mean[0], 1.9, atol=0.01)
+    assert_allclose(mean[1], 2.1, atol=0.01)
+    assert_allclose(cov, [[7e-6, 6e-6], [6e-6, 8e-6]], rtol=0.5)
+
+    # Test non-inverse sample_cdf
+    CDF = sample_cdf(normal_samples, inverse=False)
+    assert_allclose(CDF(-np.inf), 0)
+    assert_allclose(CDF(-100), 0)
+    assert_allclose(CDF(2), 0.5, atol=0.1)
+
+    samples = read_chains('./tests/example_data/pc')
+    mean2, cov2 = credibility_interval(samples["x0"], level=0.68,
+                                       weights=samples.get_weights(),
+                                       method="iso-pdf",
+                                       return_covariance=True)
+    assert_allclose(mean2[0], -0.1, atol=0.01)
+    assert_allclose(mean2[1], 0.1, atol=0.01)
+    assert_allclose(cov2, [[5e-5, 5e-5], [5e-5, 1e-4]], rtol=0.5)
```

Comment:
I think it would be cleaner to stick to the `normal_samples` for this test in `test_utils.py`, and put tests using the nested sampling example data into the test in `test_samples.py`.